DER/SecurityBasedDerivatives/EquityForwards.rdf

```diff
@@ -112,9 +133,9 @@
 				<owl:someValuesFrom rdf:resource="&fibo-fbc-fi-stl;PhysicalSettlementTerms"/>
 			</owl:Restriction>
 		</rdfs:subClassOf>
-		<rdfs:label xml:lang="en">equity forward contract</rdfs:label>
-		<skos:definition xml:lang="en">Definition needed</skos:definition>
-		<skos:editorialNote xml:lang="en">FpMl definition (unusable): A component describing an Equity Forward product.</skos:editorialNote>
+		<rdfs:label xml:lang="en">equity forward</rdfs:label>
+		<skos:definition xml:lang="en">forward contract to buy or sell the underlying equity stock, equity index, basket of equity stock, equity futures contract, or equity option at a specified future date at the price specified at the outset of the contract</skos:definition>
```

Comment:
What I found: An Equity Forward contract is an agreement between two counterparties to buy a specific number of equity stocks, stock index or basket at a given price (called strike price) at a given date.  https://finpricing.com/lib/EqFuture.html
No mention is made of the underlying being a futures contract or option.  Where did this come from?

Reply:
Hi John,

It's in the CFI standard which we looked at during our call.  That's why we raised the issue, in fact - there were two additional options including equity future and equity option listed with codes for those choices in the latest version (2019) of the ISO standard.

Reply:
Surely only "to buy" is needed, which allows for different parties in the role of buyer or seller. "Buy or sell" implies there are 2 different types of contract similar to calls or puts.